styled dashboard

tanishq-arya opened the pull request:
## Issue that this pull request solves

 Closes: #608

## Proposed changes

1. Added style for container
2. Box shadow to the cards in dashboard
3. Min max width for cards on dashboard
4. Center cards in dashboard on mobile view
 

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
![Screenshot (1348)](https://user-images.githubusercontent.com/77333275/115710615-308b8480-a390-11eb-92e8-827cb09f76e2.png)
![Screenshot (1347)](https://user-images.githubusercontent.com/77333275/115710626-32554800-a390-11eb-9d59-0900a770d6b8.png)

## Other information

Any other information that is important to this pull request

Kajol-Kumari commented on frontend/src/pages/Admin/Components/Dashboard/dashboard.module.scss:
```diff
@@ -32,7 +33,7 @@
   border-radius: 1em;
   height: auto;
-  width: 15em;
+  // width: 15em;
```
please remove this if it's not required

tanishq-arya replied:
ok

Kajol-Kumari reviewed (changes requested):
Hey @tanishq-arya the current change is looking good.
can you please try reducing the left and right margin gap on the admin panel. Maybe shift the menu card a bit more left and increase the width of the card present on the right side?

tanishq-arya commented:
> Hey @tanishq-arya the current change is looking good.
> can you please try reducing the left and right margin gap on the admin panel. Maybe shift the menu card a bit more left and increase the width of the card present on the right side?

On it

tanishq-arya commented:
![image](https://user-images.githubusercontent.com/77333275/115955819-acafd480-a516-11eb-9b50-e472716364b3.png)

Kajol-Kumari commented:
Hey @tanishq-arya thanks for incorporating the suggested changes quickly. Can we please have 3 cards in single row as there is a lot of left-out space between the cards on the right side. I believe it would look better that way.

tanishq-arya commented:
> Hey @tanishq-arya thanks for incorporating the suggested changes quickly. Can we please have 3 cards in single row as there is a lot of left-out space between the cards on the right side. I believe it would look better that way.

I'll try that and update you soon.

tanishq-arya commented:
![image](https://user-images.githubusercontent.com/77333275/115956626-7a54a600-a51b-11eb-8702-03b836a753a2.png)

Kajol-Kumari commented:
Hey @tanay13 instead of reducing the card size, can we please make the use of the left and right side left-out spaces? let's make the navigation card to stick to left(with minimal required margin) and make the right card to cover the whole right side width(just leaving the minimal margin, same as the left side). It would look consistent that way.
Please let me know if something isn't clear to you.

Kajol-Kumari commented:
Also, I can see that you have increased the width of the left-navigation card, which seems un-necessary to me.Can you please fix it?

tanishq-arya commented:
![image](https://user-images.githubusercontent.com/77333275/115992296-865b6900-a5ea-11eb-9bdc-0e068ad52dc2.png)

You were talking about something like this right? And for the left it is responsive to 25% but I added a max-width:290px.

Kajol-Kumari reviewed (changes requested):
Hey @tanishq-arya the dashboard is indeed looking good now. Can you please fix the broadcast and about-us cards that got distorted due to the changes you have made in this PR

tanishq-arya commented:
> Hey @tanishq-arya the dashboard is indeed looking good now. Can you please fix the `broadcast` and `about-us` and `faq` section cards that got distorted due to the changes you have made in this PR

Yes, sure. I was waiting for this to get approved.

Kajol-Kumari commented:
> > Hey @tanishq-arya the dashboard is indeed looking good now. Can you please fix the `broadcast` and `about-us` and `faq` section cards that got distorted due to the changes you have made in this PR
> 
> Yes, sure. I was waiting for dashboard view to get approved.

The dashboard changes are good to go. Please fix the other pages and this PR will be good to be merged :)

tanishq-arya commented:
> > > Hey @tanishq-arya the dashboard is indeed looking good now. Can you please fix the `broadcast` and `about-us` and `faq` section cards that got distorted due to the changes you have made in this PR
> > 
> > 
> > Yes, sure. I was waiting for dashboard view to get approved.
> 
> The dashboard changes are good to go. Please fix the other pages and this PR will be good to be merged :)

Will push the changes soon.

tanishq-arya commented:
![image](https://user-images.githubusercontent.com/77333275/116098345-a1040f80-a6c8-11eb-98dd-ec1847498abb.png)
@Kajol-Kumari Can you take a look at this and tell me if this size is okay?

tanishq-arya commented:
Or should I make all the min-heights same ? This looks better to me as it brings a uniformity across the cards.
![image](https://user-images.githubusercontent.com/77333275/116098858-06580080-a6c9-11eb-8647-b9a714df9d2b.png)

tanishq-arya commented:
![image](https://user-images.githubusercontent.com/77333275/116102633-80d64f80-a6cc-11eb-9139-363426af4368.png)
![image](https://user-images.githubusercontent.com/77333275/116102651-8469d680-a6cc-11eb-9f40-bd1b0ae41dc6.png)
![image](https://user-images.githubusercontent.com/77333275/116102696-8cc21180-a6cc-11eb-86bb-622802d3b347.png)
![image](https://user-images.githubusercontent.com/77333275/116103502-4620e700-a6cd-11eb-9efa-efd79bd90764.png)

tanishq-arya commented:
This is for the desktop view mobile view needs some changes.

tanishq-arya commented:
![image](https://user-images.githubusercontent.com/77333275/116107644-d57bc980-a6d0-11eb-8d29-3380d2d89f0f.png)
![Screenshot (1400)](https://user-images.githubusercontent.com/77333275/116107761-ecbab700-a6d0-11eb-8450-5213d4eaab76.png)
![Screenshot (1399)](https://user-images.githubusercontent.com/77333275/116107766-edebe400-a6d0-11eb-876b-11d1dd78f442.png)
![image](https://user-images.githubusercontent.com/77333275/116107794-f5ab8880-a6d0-11eb-852e-79f44ab43cc0.png)

Kajol-Kumari commented:
Hey @tanishq-arya! For the sections having just two cards i.e `about-us` `broadcasts` and `faq` please keep the card dimension same as before as we will be having two cards in those pages and previous card size was looking better.

tanishq-arya commented:
> Hey @tanishq-arya! For the sections having just two cards i.e `about-us` `broadcasts` and `faq` please keep the card dimension same as before as we will be having two cards in those pages and previous card size was looking better.

Okay, I'll make the changes.

tanishq-arya commented:
@Kajol-Kumari I have reverted back on the changes and added changes only to the settings one. Please check this and let me know   if anything comes up. Also should I add any media queries for the 2 column ones - faq, broadcast, about in the mobile view ?

Kajol-Kumari reviewed (approved):
Hey @tanishq-arya thanks for incorporating the suggested changes.
LGTM 👍 

Can I please get a follow-up PR for 
1. Adding card effect to `broadcast, faq and about-us` similar to the cards present on the dashboard
2. Making them responsive for all screens
3. Reducing the width of the cards on these pages a bit (Just the width)

tanishq-arya commented:
> Hey @tanishq-arya thanks for incorporating the suggested changes.
> LGTM 👍
> 
> Can I please get a follow-up PR for
> 
> 1. Adding card effect to `broadcast, faq and about-us` similar to the cards present on the dashboard
> 2. Making them responsive for all screens
> 3. Reducing the width of the cards on these pages a bit (Just the width)

Okay I ll raise a PR for that.
